docs/learn/tokenomics2/Inflation.md

```diff
@@ -2,33 +2,166 @@
+Each **cycle** consists of one or more **periods**.
+**Periods** are a core time unit in the dApp Staking protocol. Each period consists of a `Voting` and `Build&Earn` subperiods.
 
-### Stakers & dApp Rewards
+During the `Voting` subperiod, neither stakers nor dApp owners can earn any rewards, only _collators_ and _treasury_ continue earning.
+During the the `Build&Earn` subperiod, stakers earn staking rewards per **era** and have the opportunity to earn bonus reward at the end of a period.
+dApp owners also earn rewards, based on how well their dApp is performing in dApp staking, at the end of each **era**.
 
-The rewards for dApp staking, both for users and dApps, will remain unchanged in absolute terms. No immediate modifications will be implemented in this area. Future updates, including the introduction of dApp staking v3, will occur in the third phase of Tokenomics 2.0.
+### Eras
 
-We use the blockReward distribution to ensure those three requirements (and calculate them from the reward per block):
+Each **period** is divided up into multiple **eras**.
+**Era** is the core time unit in dApp staking, and its length is measured in the number of blocks.
 
-|                  | Percentage | Reward ASTR |
-|------------------|------------|-------------|
-| Block reward     | 100%       | 231.20      |
-| Treasury         | 4.78%      | 11.06       |
-| Collators        | 3.06%      | 7.07        |
-| dApp reward      | 17.27%     | 39.93       |
-| Base Staker      | 23.04%     | 53.27       |
-| Ajustable Staker | 51.84%     | 119.85      |
\ No newline at end of file
+`BuildAndEarn` subperiod consists of one or more **eras**. Each era has a fixed length.
+`Voting` subperiod always consists of exactly **one era** but its length can be longer than a standard era length.
+
+### Example
+
+* Block is produced every 12 seconds
+* **Era length** is 7200 blocks which equals 24 hours (1 day)
+* `Voting` subperiod length is 10 eras
+* `Build&Earn` subperiod length is 81 eras
+* Cycle length is **4 periods**
+
+With such configuration, we'd end up with a cycle lasting 364 days (roughly a year), and each period taking around 3 months to complete.
+
+## Recalculation
+
+When a new cycle begins, soft-inflation cap is recalculated, and according to the calculated value, rewards for all network actors are adjusted.
+
+The reason why it's a _soft-cap_, instead of a _hard-cap_ is how staker & dApp rewards are distributed.
+Rewards are only minted when they are claimed, and it is possible that at the time of reward recalculation, some rewards remain unclaimed.
+As a result, theoretically, it's possible for the cycle inflation to exceed the _soft-cap_ even though in practice it will be highly unlikely
```

Comment:
I do not get this paragraph.

The reason why it's a _soft-cap_, instead of a _hard-cap_ is how staker & dApp rewards are distributed.
Rewards are only minted when they are claimed, and it is possible that at the time of reward recalculation, some rewards remain unclaimed (and burned) thus lowering effective inflation rate.

Theoretically, due to fee burning mechanisms it is also possible for the cycle inflation to exceed the _soft-cap_ even though in practice it will be highly unlikely

Reply:
> and it is possible that at the time of reward recalculation, some rewards remain unclaimed (and burned)

This is not correct, they are not burned. They are simply unclaimed. And depending on how long we set the reward retention period, they can be claimed in the future.

The only parts that are never minted (or _burned_ if you prefer) are the:
1. unused adjustable part of the staker rewards
2. unfilled tier slots